doc/sphinx/advanced_methods.rst

```diff
@@ -118,6 +118,63 @@ The following limitations currently apply for the collision detection:
 * The ``"bind at point of collision"`` approach cannot handle collisions
   between virtual sites
 
+.. _Deleting bonds when particles are pulled:
+
+Deleting bonds when particles are pulled
+----------------------------------------
+
+With this feature, bonds between particles can be deleted automatically
+when a critical bond extension is reached. This is useful to simulate
+weak bonds between particles.
+
+The bond breakage action is specified for individual bonds via the system
+:attr:`~espressomd.system.System.bond_breakage` attribute.
+
+Several modes are available:
+
+* ``"revert_center_bond"``: delete a bond from the first particle
+* ``"revert_vs_bond"``: delete a virtual bond, can be used together with
+  the :ref:`collision detection<Creating bonds when particles collide>`
+  feature (configured in mode ``"bind_at_point_of_collision"``) to
+  reversibly create and delete a bond
+* ``"none"``: cancel an existing bond breakage specification
+
```

Comment:
We should add:
Please note that virtual sites are not automatically removed from the simulation. A simultaneous use of `"bind_at_point_of_collision"`` and breakable bonds, will sequentially always increase the particle number. If you want to remove virtual sites, you need to do so manually.

Reply:
Also: There is no special treatment implemented for treating the energy release/ consumed by bond removal. This can lead to physical inconsistencies.